ipatests/test_integration/test_commands.py

```diff
@@ -1489,3 +1489,26 @@ def test_ipa_cacert_manage_prune(self):
         self.master.run_command(['date', '-s', '-15Years'])
 
         assert isrgrootx1_nick in result
+
+    def test_client_doesnot_throw_responsenotready_error(self):
+        """
+        This testcase checks that ipa command
+        doesn't throw http.client.ResponseNotReady error
+        when current users session is deleted from the cache
+        """
+        error_msg = (
+            "http.client.ResponseNotReady: Request-sent \n"
```

Comment:
Is there really a space at the end of the string when the issue happens?

Reply:
@flo-renaud i did see the space at the end of the string.